nservicebus/messaging/transactions.md

```diff
@@ -118,12 +106,10 @@ NOTE: This requires the selected storage to support enlisting in transaction sco
 
 WARNING: This might escalate to a distributed transaction if data in different databases are updated.
 
-
 ## Controlling transaction scope options
 
 The following options for transaction scopes used during message processing can be configured.
 
```

Comment:
Why the whitespace removal. Some spacing between headers makes the raw md much more readable